README-hi.md

```diff
@@ -67,7 +70,9 @@ make serve
 
 ## समुदाय, चर्चा, योगदान और समर्थन
 
-[Community page](http://kubernetes.io/community/) पर कुबरनेट्स समुदाय के साथ जुड़ना सीखें।
+[Community page](http://kubernetes.io/community/) पर कुबरनेट्स सम
+
+ुदाय के साथ जुड़ना सीखें।
```

Comment:
@pojith I dont think here we need to give one line blank with half of the incomplete word. 
I think it is not needed. Can you explain the intention of doing so?